great_expectations/rule_based_profiler/data_assistant/data_assistant_dispatcher.py

```diff
@@ -29,24 +29,27 @@ def __init__(self, data_context: "BaseDataContext") -> None:  # noqa: F821
     def __getattr__(self, name: str) -> DataAssistantRunner:
         # Both, registered data_assistant_type and alias name are supported for invocation.
 
-        # Attempt to utilize data_assistant_type name for invocation first.
-        data_assistant_cls: Optional[
-            Type[DataAssistant]
-        ] = DataAssistantDispatcher.get_data_assistant_impl(data_assistant_type=name)
+        data_assistant_cls: Optional[Type[DataAssistant]] = None
 
-        # If no registered data_assistant_type exists, try aliases for invocation.
-        if data_assistant_cls is None:
-            type_: Type[DataAssistant]
-            for type_ in DataAssistantDispatcher._registered_data_assistants.values():
-                if type_.__alias__ == name:
-                    data_assistant_cls = type_
-                    break
+        # Attempt to utilize alias for invocation first.
+        type_: Type[DataAssistant]
+        for type_ in DataAssistantDispatcher._registered_data_assistants.values():
```

Comment:
I don't get what's happening here. The `_registered_data_assistants` is a dictionary, we should use it as one. Doesn't the key match the `name` parameter?

Reply:
oh wow wait this makes a ton of sense. I originally did not register the aliases in the dict so I wasn't aware we had access to them at this point.

We should now be able to just use `get()` and search through aliases and full names at the same time. Thanks for calling this out!